src/celeritas/optical/ScintillationInput.hh

```diff
@@ -0,0 +1,40 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/optical/ScintillationInput.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "celeritas/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Input data for sampling optical photons for a given scintillation step.
+ */
+struct ScintillationInput
```

Comment:
These data look identical to the cerenkov one that @amandalund  added: perhaps we can just rename it to `OpticalDistributionData` and reuse it here?

Reply:
Renamed CerenkovDistributionData to `OpticalDistributionData` and used it for ScintillationInput (which is now removed).